src/app/_styles/tooltip.css

```diff
@@ -0,0 +1,7 @@
+.tooltip-content {
+  @apply max-w-xs rounded-lg bg-brand-200 px-4 py-3 text-sm leading-tight text-brand-800 will-change-[transform,opacity];
+}
+
+.tooltip-arrow {
+  @apply fill-brand-200;
+}
```

Comment:
Why is this abstracted?

Reply:
It was initially abstracted because we had two tooltip components to keep the style in one place.

> I think we should migrate the existing CSS code currently in addComponents to individual CSS files.

We already have a few style abstractions, such as `brand-outline`, in `addComponents` in the Tailwind Config. But I find individual CSS classes easier to write and read as they use the same Tailwind syntax, versus the CSS-in-JS syntax.

```ts
    plugin(function addBrandOutline({ addComponents, theme }) {
      addComponents({
        '.brand-outline': {
          outlineStyle: 'solid',
          outlineColor: theme('colors.brand.100'),
          outlineWidth: 2,
          outlineOffset: 0,
          borderColor: 'transparent',
        },
        '.row': {
          height: 52,
        },
        '.cell': {
          paddingLeft: theme('padding.5'),
          paddingRight: theme('padding.5'),
          textAlign: 'left',
          textWrap: 'nowrap',
          fontWeight: '400',
          '&:first-child': {
            paddingLeft: theme('padding.10'),
          },
          '&:last-child': {
            paddingRight: theme('padding.10'),
          },
        },
      })
    }),
```

Reply:
Move the style into the component

Reply:
✅